kubectl-fdb/cmd/remove_process_group.go

```diff
@@ -117,62 +126,83 @@ kubectl fdb -n default remove process-group -c cluster --process-class="stateles
 	return cmd
 }
 
-// replaceProcessGroups adds process groups to the removal list of their respective clusters.
+type replaceProcessGroupsOptions struct {
+	clusterLabel      string
+	processClass      string
+	withExclusion     bool
+	wait              bool
+	removeAllFailed   bool
+	useProcessGroupID bool
+}
+
+// replaceProcessGroups adds process groups to the removal list of their respective clusters, and returns a count of
+// the number of removed processGroups (pods) and any encountered error.
 // If clusterName is specified, it will ONLY do so for the specified cluster.
 // If processClass is specified, it will ignore the given ids and remove all processes in the given cluster whose pods
 // have a processClassLabel matching the processClass.
-func replaceProcessGroups(kubeClient client.Client, clusterName string, ids []string, namespace string, clusterLabel string, processClass string, withExclusion bool, wait bool, removeAllFailed bool, useProcessGroupID bool) error {
-	if len(ids) == 0 && !removeAllFailed && processClass == "" {
-		return nil
+func replaceProcessGroups(cmd *cobra.Command, kubeClient client.Client, clusterName string, ids []string, namespace string, opts replaceProcessGroupsOptions) (int, error) {
```

Comment:
I don't love returning the total here but without it we can't get a removal total for cordon.  I think named return parameters could be helpful here, but I know some people don't like them.